stakingModel/contracts/UBIScheme.sol

```diff
@@ -247,29 +228,28 @@ contract UBIScheme is AbstractUBI {
         return _claim(msg.sender);
     }
 
-    /* @dev In Order to update users from active to inactive, we give out incentive to people
-     * to update the status of inactive users, this action is called “Fishing”. Anyone can
-     * send a tx to the contract to mark inactive users. The “fisherman” receives a reward
+    /* @dev In order to update users from active to inactive, we give out incentive to people
+     * to update the status of inactive users, this action is called "Fishing". Anyone can
+     * send a tx to the contract to mark inactive users. The "fisherman" receives a reward
      * equal to the daily UBI (ie instead of the “fished” user). User that “last claimed” > 14
-     * can be “fished” and made inactive (reduces active users count by one). Requires
+     * can be "fished" and made inactive (reduces active users count by one). Requires
      * contract to be active.
      * @param account to fish
      * @return A bool indicating if UBI was fished
      */
     function fish(address account)
-        public
+        public // TODO NOT TOO MANY TESTS, NOT CHECKING SO MANY THINGS IN EACH TEST, LOOK AT ETORO X
         requireActive
         returns (bool)
     {
         setDay();
-        require(identity.isWhitelisted(account), "is not whitelisted");
         require(!isNewUser(account), "a new user");
-        require(checkActiveEntitlement(account), "an active user");
-        require(activeUsersAddresses[account], "has already fished");
+        require(!isPendingUser(account), "a pending user");
```

Comment:
redundant test. the only thing we care about is if user is active or not

Reply:
no longer pending list. there is a registered user, active user, inactive user. when a new user claims for the first time he become to registered and active. if the user had not been active for 14 days he becomes inactive but he still registered. there are 2 cases: 
1. the user has been fished -> the user becomes to unregistered and added to the fished user list and the user removed from the active user count. -> the user claims and added to the active user count. 
2. the user claim but since he was inactive for 14 days he will receive his funds on the day after. since the user marks as a registered user the active count won't change.

3a130df16ce6b41f40192c76921b58e63d2c5ccd

Reply:
2. if user wasnt fished he should be able to claim as usual no need for day after

Reply:
updated
ebf3c7359855b23286dfcd1533f5e4fc3f479521